src/reducers/searchReducer.js

```diff
@@ -45,7 +45,7 @@ const initialState = {
   dateCreated: "",
   query_fields:"All",
   sort_by_published_date_order:"relevance",
-  partialMatch:"true"
```

Comment:
@ianlongden I don't nkow how this works, sometimes a string of 'true' needs to be passed, but I don't know search.

Reply:
Partial Match is what adds a wildcard after a users search.  The curators wanted this on by default (which is why its in the reducer).  This line still needs to be in the reducer.